src/components/Notifications/Notification.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) [2020] SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { actionCreators as actions } from '../../state/ducks/notifications';
+
+function Notification({ notification }) {
+  const dispatch = useDispatch();
+  const { id, text } = notification;
+
+  return (
+    <div
+      role="alertdialog"
+      className="w-1/3 mb-2 p-3 text-sm text-gray-700 border-b-2 border-secondary bg-white shadow-lg"
+    >
+      {text}
+
+      <div className="mt-2">
+        <button className="text-blue-600 font-bold mr-2" onClick={() => dispatch(actions.close(notification.id))}>
+          Dismiss
+        </button>
+
+        {notification.type &&
```

Comment:
I guess we can refine this in the future, not sure whether having type or not is the right criteria for allowing the "do not show again".

Reply:
Yes, you are right. I came to the same conclusion. But now that I've added support for notification actions, this code is not relevant anymore :)